plugins/action/zos_copy.py

```diff
@@ -124,9 +125,6 @@ def run(self, tmp=None, task_vars=None):
         #     msg = "Invalid port provided for SFTP. Expected an integer between 0 to 65535."
         #     return self._exit_action(result, msg, failed=True)
 
```

Comment:
In the prior release the code did some checks to see if the source and destination existed  when force was not true to avoid overwriting a users content accidentally. Maybe the logic or check has been put in the module, i have not gotten to the module yet but want to at at least note it here so i could resolve it later.